### DISCUSSION TITLE
SSR: `getStaticPaths()` and `getStaticProps()`

### DISCUSSION THREAD

**Discussion by GabenGar** [i18next/next-i18next#1755] [Q&A] [answered]
Do I have to add `getStaticPaths` and `getStaticProps` on every page to benefit from server-rendered internationalization?
Also is there a way to have some sort of default namespaces which come bundled with `serverSideTranslations()` call and the second argument only extending this list?

**Comment by isaachinman** [comment, marked as answer]
For your first question, please see https://github.com/isaachinman/next-i18next/issues/1215.

For your second question, the answer is no, outside of [`defaultNS`](https://www.i18next.com/overview/configuration-options#languages-namespaces-resources).
